.github/workflows/auto-jdk-matrix.yml

```diff
@@ -0,0 +1,67 @@
+name: DataSketches-Memory Auto JDK Matrix Test & Install
+
+on:
+    pull_request:
+    push:
+    #    branches: [ master ]
+    workflow_dispatch:
+
+env:
+    MAVEN_OPTS: -Xmx4g -Xms1g
```

Comment:
Do we actually need 4g??

Reply:
Oh jees, that has been there for a long time, I don't remember why I made it 4g at the time.